app/main.py

```diff
@@ -1,13 +1,22 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors":
```

Comment:
your solution looks good but your code style doesn't. Read this checklist to fix errors:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md
In this line, it is the rule that `The curly braces must be located in one of two options: open and start with the text or have a line break between the text.`